search/include/pcl/search/impl/organized.hpp

```diff
@@ -350,14 +350,14 @@ pcl::search::OrganizedNeighbor<PointT>::estimateProjectionMatrix ()
   std::vector<int> indices;
   indices.reserve (input_->size () >> (pyramid_level_ << 1));
   
-  for (unsigned yIdx = 0, idx = 0; yIdx < input_->height; yIdx += ySkip, idx += input_->width * (ySkip - 1))
+  for (unsigned yIdx = 0, idx = 0; yIdx < input_->height; yIdx += ySkip, idx += input_->width * ySkip)
   {
-    for (unsigned xIdx = 0; xIdx < input_->width; xIdx += xSkip, idx += xSkip)
+      for (unsigned xIdx = 0, idx2 =idx; xIdx < input_->width; xIdx += xSkip, idx2 += xSkip)
```

Comment:
Please remove extra two spaces before `for` and add a space after equal sign in `=idx`.